lib/Site.js

```diff
@@ -250,8 +264,12 @@ Site.prototype.createPage = function (config) {
                                path.join(this.siteAssetsDestPath, 'css', 'github.min.css')),
       markbind: path.relative(path.dirname(resultPath),
                               path.join(this.siteAssetsDestPath, 'css', 'markbind.css')),
+      siteNavCss: path.relative(path.dirname(resultPath),
```

Comment:
Move this down to maintain alphabetical order.

Reply:
Order was decided from file type (`.css`) first, then name.